types/hooks/screens/useAnonymousChatInfoScreenHook.types.ts

```diff
@@ -1,9 +1,23 @@
 import {ChannelList, ChannelListMemberSchema} from '../../database/schema/ChannelList.types';
+import {Member} from '../../database/schema/ChatListDetail.types';
 
 interface UseAnonymousChatInfoScreenHook {
   channelInfo: ChannelList;
   goBack: () => void;
-  onContactPressed: (item: ChannelListMemberSchema) => void;
+  onContactPressed: (item: ChannelListMemberSchema, from?: string) => void;
+  isAnonymous: boolean;
+  selectedUser: ChannelListMemberSchema;
+  showPopupBlock: boolean;
+  handleClosePopup: () => void;
+  isAnonymousUser: (item: any) => boolean;
+  handlePressPopup: (status: string) => void;
+  handleCloseSelectUser: () => void;
+  openModal: boolean;
+  isAnonymousModalOpen: boolean;
+  blockModalRef: () => void;
+  handleShowArrow: (item: Member) => void;
+  goToEditGroup: () => void;
+  loadingChannelInfo: boolean;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made to the `UseAnonymousChatInfoScreenHook` interface seem to be adding more functionality and control over the chat screen. However, there's a potential issue with the `isAnonymousUser` function. The parameter `item` is typed as `any`, which can lead to potential type errors or misuse of this function. It would be better to provide a specific type for `item` if possible.
```diff
-   isAnonymousUser: (item: any) => boolean;
+   isAnonymousUser: (item: SpecificType) => boolean;
```
Please replace `SpecificType` with the actual type that `item` should be. If `item` can be multiple types, consider using a TypeScript union type.

Also, it's not clear what the `status` string in `handlePressPopup` function represents. If there are only a limited number of valid strings that `status` can take, consider using a TypeScript string literal type or an enum to make the code safer and easier to understand.
```diff
-   handlePressPopup: (status: string) => void;
+   handlePressPopup: (status: 'status1' | 'status2' | 'status3') => void;
```
Replace `'status1'`, `'status2'`, and `'status3'` with the actual status strings. If there are many possible status strings, consider using an enum.


<!-- This is an auto-generated comment by OSS CodeRabbit -->